.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,16 @@
+name: CompatHelper
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
```

Comment:
I believe CompatHelper and TagBot now needs an update due to how GitHub handles default permission, see also https://github.com/invenia/PkgTemplates.jl/pull/347

Reply:
CompatHelper needs write content permission upate, although I don't see changes in TagBot.yml in PkgTemplates.jl

Reply:
Ohhh! I believe I made some mistakes here -- the write content permission is required only if we're using GITHUB_TOKEN.

Thus here as long as we set up ssh key `DOCUMENTER_KEY`, we're good.

> https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
> You can use permissions to modify the default permissions granted to the GITHUB_TOKEN, adding or removing access as required, so that you only allow the minimum required access